packages/keystone/lib/Keystone/index.js

```diff
@@ -34,6 +33,9 @@ const List = require('../List');
 const { DEFAULT_DIST_DIR } = require('../../constants');
 const { CustomProvider, ListAuthProvider, ListCRUDProvider } = require('../providers');
 
+// composePlugins([f, g, h])(o, e) = h(g(f(o, e), e), e)
+export const composePlugins = fns => (o, e) => fns.reduce((acc, fn) => fn(acc, e), o);
```

Comment:
Can we move this down to line 310? It's only used in that one specific place and it's much easier to understand the code when you can see it all in one place. Also, there's no need to be exporting this function.